### DISCUSSION TITLE
Something went wrong

### DISCUSSION THREAD

**Discussion by othyn** [community/community#29203] [Mobile]
Almost every panel I try to load, this error is constant. Just 'Something went wrong.' Nothing works in the app!

Things I've tried;

- Clearing cache
- Signing in/out
- Deleting and re-downloading the app, making sure to delete all app contents and docs
- Using different networks; WiFi, cellular, other WiFi networks...

I think there is something wrong with my account or GitHub's API as no matter how many times I clear things out, networks I'm on or anything, nothing gets it working.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 12 Pro Max (iOS 16.0)
</details>

And before people mention the iOS 16 beta, it's not the beta. I've had this issue for a _very_ long time. I've just given up using the app and every now and again try it only to be disappointed again.

**EDIT**:

For future readers, if you are experiencing this problem, I contacted GitHub support in the end as it was outside of the scope of this message board. An engineer patched something I believe on my account (given the back and forth we had) and it started working again shortly after. No idea what the root cause is, but its not something that can be fixed 'client' side.

**Comment by haldun** [comment]
Hi @othyn, I am sorry to hear that you are having problems with the app. In order to pin down the problem, can you please share some details about the issues you are having? Is that happens on every screen?

**Reply by othyn** [reply]
Absolutely!

Unfortunately I don't have any steps to reproduce as its just happening no matter what.

What is very interesting though is that I've had friends and collegues try to visit my account (Othyn) in the GitHub app and they get the same error when trying to view or load any of my repo's. However, this problem does not exist on the web version of GitHub.

## Noticable behaviour

- Any page I try to access that is related to my own account's (Othyn) repo's results in the request timing out and being presented with a page that simply states 'Something went wrong.' with the option to 'Try again', which ultimately does nothing but result in the same error page. This behaviour is still observed even after doing all of the following:
    - Clearing cache
    - Signing in/out
    - Deleting and re-downloading the app, making sure to delete all app contents and docs
    - Using different networks; WiFi, cellular, other WiFi networks...
- This seems to only affect access to Repo's that my account owns.
- I **can** access other accounts, organisations and shared repo's just fine, I **cannot** access any of my own repo's through the GitHub app. GitHub web is unaffected.
- Friends and collegues GitHub app's **also** exhibit the same error when viewing any of my account's (Othyn) repo's. But this does **not** affect their own accounts in the GitHub app or viewing my account's (Othyn) repo's on the web.

## Screenshots

### When trying to interact with the 'favorite repo' functionality, any search results that contain my own repo's makes the request fail with this page, although I can search and add other repo's as long as the rendered list does not contain any repo's I own. However this will just randomly break completely at times too and not render any repos at all resulting in the same error until I quit and restart the app.

<div align="center">
    <img src="https://user-images.githubusercontent.com/7256684/185089835-42e2c384-bd6f-405c-93a1-9f8ff8c77013.PNG" alt="favorites repo screenshot" width="200" height="auto" />
</div>

### When trying to *view any* of my accounts repos, it just flat out fails.

<div align="center">
    <img src="https://user-images.githubusercontent.com/7256684/185089829-dbb38415-a2d5-4a44-b67f-e493e386840a.PNG" alt="trying to open any of my repos" width="200" height="auto" />
</div>

### When trying to *open any* of my accounts repos, it just flat out fails.

<div align="center">
    <img src="https://user-images.githubusercontent.com/7256684/185089821-7baa17a3-d0b9-4036-92bd-f86abfeea737.PNG" alt="viewing my accounts repos" width="200" height="auto" />
</div>

**Reply by othyn** [reply]
I've also sent off a ticket to GitHub as I think this is out of the communities hands tbf.

**Reply by haldun** [reply]
Thank you very much for the details. I can actually reproduce this when I navigate to one of your repositories. I will create an issue for this to track further.

**Reply by othyn** [reply]
Amazing, thanks!

**Reply by Fopen1** [reply]
У меня такая же ошибка в веб версии

**Comment by thorwhalen** [comment]
Was this solved?
How was it solved?

Because I have the same issue.
I'm on an iPhone (ne newest I think). The issue appeared first a few months ago I think.

**Reply by othyn** [reply]
I contacted GitHub support in the end as it was outside of the scope of this. An engineer patched something I believe on my account (given the back and forth we had) and it started working again.

**Comment by rocheparadox** [comment]
I faced the same issue in my iPhone and iPad. It is not fixed yet. However, I found a workaround. If you click your **profile** (bottom right in *iPhone*) or **your profile** (bottom left in *iPad*) and then click the **repositories** from the options, the repositories are fetched and displayed properly.

**Comment by riteshjagga** [comment]
I faced the same issue in my iPhone. I contacted GitHub support but they couldn't fix it and shared this note

> Note that while this support ticket may automatically close, that the internal issue will remain open and under investigation.

However, I could get it working on my own by following these steps:

**Note:** The steps involve Signing out and Signing in. I remember that I did that in the past before I raised the ticket with GitHub support but this time I think I got something different on signing out.

 - Navigate to **Profile (In the bottom-right) > Settings (Cog icon button) > Manage Accounts > Edit > Delete (on the left)  > Sign out (on the right)**
 - A popup opened saying that "We couldn't sign out you successfully because some token was not found" and options to OK (which is sort of Cancel) and "Sign out anyway".
 - I said OK once without paying attention carefully and this time I choose "Sign out anyway"
 - Signed in again to see that I could access Repositories and Pull Requests.

**Reply by keypointt** [reply]
Worked for me. Thank you.

**Comment by kumaraguru1735** [comment]
I am having same issue  for the past week, still not cleared

**Reply by Fopen1** [reply]
У меня год уже бесить этот ошибка не исправляют

**Comment by alex-ak1** [comment]
Got same error on old browser inside virtual machine (firefox 86, ubuntu 20). Switched to new google chrome and all works fine.

**Comment by Fopen1** [comment]
оказывается русский язык не поддерживает гитхаб

**Comment by dataspun** [comment]
Same issue with GitHub iOS app version 1.198.0  
Repos are fine but not the org -- I'm an owner.  
Workarounds above (signing out/in) don't work.

Update: 
Working with GitHub support
Using the beta app to help developer team troubleshoot

**Reply by damoncoo** [reply]
Same issue with the latest iOS app

**Comment by MichalecSS** [comment]
Michal Bičík

**Comment by knaz112128** [comment]
> Almost every panel I try to load, this error is constant. Just 'Something went wrong.' Nothing works in the app!
> 
> 
> 
> Things I've tried;
> 
> 
> 
> - Clearing cache
> 
> - Signing in/out
> 
> - Deleting and re-downloading the app, making sure to delete all app contents and docs
> 
> - Using different networks; WiFi, cellular, other WiFi networks...
> 
> 
> 
> I think there is something wrong with my account or GitHub's API as no matter how many times I clear things out, networks I'm on or anything, nothing gets it working.
> 
> 
> 
> <details>
> 
> <summary>ℹ️ Additional Debug Info</summary>
> 
> GitHub for iOS, Version 1.75.0 (74400986)
> 
> Device: iPhone 12 Pro Max (iOS 16.0)
> 
> </details>
> 
> 
> 
> And before people mention the iOS 16 beta, it's not the beta. I've had this issue for a _very_ long time. I've just given up using the app and every now and again try it only to be disappointed again.
> 
> 
> 
> **EDIT**:
> 
> 
> 
> For future readers, if you are experiencing this problem, I contacted GitHub support in the end as it was outside of the scope of this message board. An engineer patched something I believe on my account (given the back and forth we had) and it started working again shortly after. No idea what the root cause is, but its not something that can be fixed 'client' side.

**Reply by knaz112128** [reply]
// This file is not on TypeScript so that it's reused in the Node scripts

const statuses = {
  100: { code: 100, message: 'Continue' },
  101: { code: 101, message: 'Switching Protocols' },
  102: { code: 102, message: 'Processing' },
  103: { code: 103, message: 'Early Hints'},
  200: { code: 200, message: 'OK' },
  201: { code: 201, message: 'Created' },
  202: { code: 202, message: 'Accepted' },
  203: { code: 203, message: 'Non-Authoritative Information' },
  204: { code: 204, message: 'No Content' },
  206: { code: 206, message: 'Partial Content' },
  205: { code: 205, message: 'Reset Content' },
  207: { code: 207, message: 'Multi-Status' },
  208: { code: 208, message: 'Already Reported' },
  214: { code: 214, message: 'Transformation Applied' },
  226: { code: 226, message: 'IM Used' },
  300: { code: 300, message: 'Multiple Choices' },
  301: { code: 301, message: 'Moved Permanently' },
  302: { code: 302, message: 'Found' },
  303: { code: 303, message: 'See Other' },
  304: { code: 304, message: 'Not Modified' },
  305: { code: 305, message: 'Use Proxy' },
  307: { code: 307, message: 'Temporary Redirect' },
  308: { code: 308, message: 'Permanent Redirect' },
  400: { code: 400, message: 'Bad Request' },
  401: { code: 401, message: 'Unauthorized' },
  402: { code: 402, message: 'Payment Required' },
  403: { code: 403, message: 'Forbidden' },
  404: { code: 404, message: 'Not Found' },
  405: { code: 405, message: 'Method Not Allowed' },
  406: { code: 406, message: 'Not Acceptable' },
  407: { code: 407, message: 'Proxy Authentication Required' },
  408: { code: 408, message: 'Request Timeout' },
  409: { code: 409, message: 'Conflict' },
  410: { code: 410, message: 'Gone' },
  411: { code: 411, message: 'Length Required' },
  412: { code: 412, message: 'Precondition Failed' },
  413: { code: 413, message: 'Payload Too Large' },
  414: { code: 414, message: 'Request-URI Too Long' },
  415: { code: 415, message: 'Unsupported Media Type' },
  416: { code: 416, message: 'Request Range Not Satisfiable' },
  417: { code: 417, message: 'Expectation Failed' },
  418: { code: 418, message: 'I’m a teapot' },
  419: { code: 419, message: 'Page Expired' },
  420: { code: 420, message: 'Enhance Your Calm' },
  421: { code: 421, message: 'Misdirected Request' },
  422: { code: 422, message: 'Unprocessable Entity' },
  423: { code: 423, message: 'Locked' },
  424: { code: 424, message: 'Failed Dependency' },
  425: { code: 425, message: 'Too Early' },
  426: { code: 426, message: 'Upgrade Required' },
  428: { code: 428, message: 'Precondition Required' },
  429: { code: 429, message: 'Too Many Requests' },
  431: { code: 431, message: 'Request Header Fields Too Large' },
  444: { code: 444, message: 'No Response' },
  450: { code: 450, message: 'Blocked by Windows Parental Controls' },
  451: { code: 451, message: 'Unavailable For Legal Reasons' },
  495: { code: 495, message: 'SSL Certificate Error' },
  496: { code: 496, message: 'SSL Certificate Required' },
  497: { code: 497, message: 'HTTP Request Sent to HTTPS Port' },
  498: { code: 498, message: 'Token expired/invalid' },
  499: { code: 499, message: 'Client Closed Request' },
  500: { code: 500, message: 'Internal Server Error' },
  501: { code: 501, message: 'Not Implemented' },
  502: { code: 502, message: 'Bad Gateway' },
  503: { code: 503, message: 'Service Unavailable' },
  504: { code: 504, message: 'Gateway Timeout' },
  506: { code: 506, message: 'Variant Also Negotiates' },
  507: { code: 507, message: 'Insufficient Storage' },
  508: { code: 508, message: 'Loop Detected' },
  509: { code: 509, message: 'Bandwidth Limit Exceeded' },
  510: { code: 510, message: 'Not Extended' },
  511: { code: 511, message: 'Network Authentication Required' },
  521: { code: 521, message: 'Web Server Is Down' },
  522: { code: 522, message: 'Connection Timed Out' },
  523: { code: 523, message: 'Origin Is Unreachable' },
  525: { code: 525, message: 'SSL Handshake Failed' },
  530: { code: 530, message: 'Site Frozen' },
  599: { code: 599, message: 'Network Connect Timeout Error' },
}

module.exports = statuses;
